dockerfile/make-ready/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM fedora:rawhide
```

Comment:
In the future we probably want to turn this into a variable, but let's just start with `:rawhide` indeed.

Reply:
You can have variables in the Dockefile?

Reply:
Yes, with [ARG](https://docs.docker.com/engine/reference/builder/#arg). A caveat is that they don't yet work in `FROM` on older docker versions (such as RHEL 7), but they work fine with podman.